spec/index.bs

```diff
@@ -6136,14 +6137,15 @@ Completion of a fence is signaled from the {{GPUQueue}} that created it.
 <dl dfn-type=method dfn-for=GPUQueue>
     : <dfn>signal(fence, signalValue)</dfn>
     ::
+        Signals the given fence, increasing it's completed value to `signalValue`.
```

Comment:
nit: Not sure what the best way will be to write the prose for all of the commands, but probably something like "Enqueues a command which signals..."

Reply:
Fine by me, though that implies that we need to do a pass of the whole spec to reflect that language. Follow up PR?

Reply:
Definitely not now. We should eventually figure out how we should write these, with some things in mind:
- Somewhere it should be clear that the functions on the encoders just add some data onto the end of a list, and in `submit` we actual execute the commands in that list.
- We need to put the "how to execute the command" algorithm somewhere. Perhaps it's just inside the function definitions, in the form of a closure? like "Enqueue an item onto the encoder which, when executed, performs the following tasks"

Not urgent though.